dotenv/mod.ts

```diff
@@ -35,6 +36,7 @@
  * GREETING=hello world
  * ```
```

Comment:
problem here again; section will be part of previous example

Reply:
Should `sh` code blocks include the `@example` tag?

Reply:
i feel that it doesnt make too much sense to make these examples, as it is explaining how to use & its features it rather than standalone example